test/node/error_test.js

```diff
@@ -23,10 +29,15 @@ describe('BSONTypeError', function () {
 describe('BSONError', function () {
   it('should evaluate true on instanceof BSONError and Error', function () {
     const bsonErr = new BSONError();
-    expect(bsonErr instanceof BSONError).to.be.true;
-    expect(bsonErr instanceof Error).to.be.true;
-    expect(bsonErr).to.be.instanceOf(BSONError);
-    expect(bsonErr).to.be.instanceOf(Error);
+    if (__isWeb__) {
+      expect(bsonErr).to.have.property('name', 'BSONError');
```

Comment:
`instanceof` doesn't work since the "web" tests are run in a different context, so name prop check will suffice